README.md

```diff
@@ -446,6 +447,34 @@ permission.update() # Update!!
 
 ```
 
+## Token
+https://www.jfrog.com/confluence/display/RTF5X/Access+Tokens#AccessTokens-RESTAPI
+```python
+from requests.auth import HTTPBasicAuth
+from artifactory import ArtifactoryPath
+from dohq_artifactory import Token
+
+session = ArtifactoryPath('https://artifactory_dns/artifactory', auth=('admin', 'admin_password'), auth_type=HTTPBasicAuth, verify=False)
+
+# Read token for readers group
+group_name="readers"
+scope = "api:* member-of-groups:"+group_name
+token = Token(session,  scope=scope)
+token.read()
+
+# Create token for member of the readers
+group_name="readers"
+scope = "api:* member-of-groups:"+group_name
+subject=group_name
+token = Token(session, scope=scope, username=subject, expires_in=31557600, refreshable=True)
+response=token.create()
+
+print("Readonly token:")
+print("Username: "+token.username)
+print("Token: "+token.token['access_token'])
```

Comment:
Could we use this access token in this library? It will be helpful if you give an example